### ISSUE TITLE
[Bug]: no visible focus on sp-top-nav-item

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#2071]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-top-nav sp-top-nav-item

### Expected behavior

All components should have a visible focus when they receive focus

### Actual behavior

There is no visible focus

### What browsers are you seeing the problem in?

Firefox, Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/top-nav/
2. Tab through the page until you get to the top nav example
3. Note there is no visible focus

### Sample code that illustrates the problem

fonts.adobe.com

**Comment by Westbrook** [comment]
That's awkward as it seems to work in _some_ contexts: https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/top-nav--default This might be due to tricks of the `:focus-visible` polyfill that's hanging around the library for just a little longer. Thanks for the catch, we'll figure out what needs to change to guarantee this!

**Comment by jnurthen** [comment]
@Westbrook didn't file yet but also saw some really weird behaviour with top-nav with Firefox

**Comment by Westbrook** [comment]
Was it along the lines of it follow macOS rules and not tab stopping on `<a href="#">` elements, or something else?
